src/V3String.cpp

```diff
@@ -20,6 +20,7 @@
 #include "V3String.h"
 
 #include "V3Error.h"
+#include "V3FileLine.h"
```

Comment:
V3String is supposed to be low-level routines so I'd prefer not to have a FileLine dependence (and the related mess with VL_COVERAGE).

Choices are to move unquoteSVString perhaps to V3File to match the corresponding inverse, or have it return a error string that the caller handles.

Reply:
V3File is dedicated to file IO and content generation, so still put in V3String, but return an error message instead of using FileLine